token-lending/program/src/processor.rs

```diff
@@ -449,6 +450,41 @@ fn process_deposit_reserve_liquidity(
     let clock = &Clock::from_account_info(next_account_info(account_info_iter)?)?;
     let token_program_id = next_account_info(account_info_iter)?;
 
+    // We don't care about the return value here, so just ignore it.
+    let _ = match _process_deposit_reserve_liquidity(
```

Comment:
so like wouldn't this be the same as doing 
```
let _process_deposit_reserve_liquidity(...)?;
Ok(())
```

Reply:
oh yeah that seems better